### PR TITLE
[CI:DOCS] Fix documentation on read-only-tmpfs

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by rhatdan** [comment]
@edsantiago PTAL

**Review by edsantiago — APPROVED** [review]
LGTM apart from the one-long-line format; I won't block over that. Thank you.

File: docs/source/markdown/podman-run.1.md.in [hunk 1]
```diff
@@ -470,9 +470,7 @@ content. Installing packages into _/usr_, for example. In production,
 applications seldom need to write to the image.  Container applications write
 to volumes if they need to write to file systems at all. Applications can be
 made more secure by running them in read-only mode using the **--read-only** switch.
-This protects the container's image from modification. Read-only containers may
-still need to write temporary data. The best way to handle this is to mount
-tmpfs directories on _/run_ and _/tmp_.
+This protects the container's image from modification. By default read-only containers can write to temporary data. Podman mounts a tmpfs on /run and /tmp within the container. If the container should not write to any file system within the container, including tmpfs, set --read-only-tmpfs=false.
```
Comment (edsantiago) [inline docs/source/markdown/podman-run.1.md.in:473]: On second thought, though, you might want to backtick the paths, and asteriskize the option.

**Comment by edsantiago** [comment]
LGTM, but documentation-focused reviewers may still want the double asterisks on the `--option`.

**Comment by rhatdan** [comment]
@TomSweeneyRedHat PTAL

**Pull request merged**